module/os/freebsd/zfs/sysctl_os.c

```diff
@@ -779,7 +779,7 @@ param_set_min_auto_ashift(SYSCTL_HANDLER_ARGS)
 
 /* BEGIN CSTYLED */
 SYSCTL_PROC(_vfs_zfs, OID_AUTO, min_auto_ashift,
-	CTLTYPE_U64 | CTLFLAG_RWTUN | CTLFLAG_MPSAFE,
+	CTLTYPE_UINT | CTLFLAG_RWTUN | CTLFLAG_MPSAFE,
 	&zfs_vdev_min_auto_ashift, sizeof (zfs_vdev_min_auto_ashift),
 	param_set_min_auto_ashift, "QU",
```

Comment:
U64->UINT, shouldn't QU -> U ?

Reply:
Nice catch. Looking at other `SYSCTL_PROC()` invocations, we should be using `"IU"`. I just fixed it.